src/rpc/getPublicKey.ts

```diff
@@ -0,0 +1,16 @@
+import {Wallet} from "../interfaces";
+import {generateKeys} from "../crypto/keys";
+import {iterableToHexString} from "../util";
+
+export async function getPublicKey(wallet: Wallet): Promise<string> {
+  const keyPairState = wallet.getPluginState();
+  if (keyPairState != null && keyPairState.polkadot.account != null) {
```

Comment:
`!keyPairState && !keyPairState.polkadot.account` so you are chcking both null and undefined